[2023-10-22T08:38:34.341+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_day18.fetch_data_from_psql manual__2023-10-22T08:38:23.431099+00:00 [queued]>
[2023-10-22T08:38:34.356+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_day18.fetch_data_from_psql manual__2023-10-22T08:38:23.431099+00:00 [queued]>
[2023-10-22T08:38:34.356+0000] {taskinstance.py:1359} INFO - Starting attempt 1 of 1
[2023-10-22T08:38:34.378+0000] {taskinstance.py:1380} INFO - Executing <Task(PythonOperator): fetch_data_from_psql> on 2023-10-22 08:38:23.431099+00:00
[2023-10-22T08:38:34.386+0000] {standard_task_runner.py:57} INFO - Started process 476 to run task
[2023-10-22T08:38:34.391+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'etl_day18', 'fetch_data_from_psql', 'manual__2023-10-22T08:38:23.431099+00:00', '--job-id', '164', '--raw', '--subdir', 'DAGS_FOLDER/ETL_day18.py', '--cfg-path', '/tmp/tmpbys316av']
[2023-10-22T08:38:34.397+0000] {standard_task_runner.py:85} INFO - Job 164: Subtask fetch_data_from_psql
[2023-10-22T08:38:34.465+0000] {task_command.py:415} INFO - Running <TaskInstance: etl_day18.fetch_data_from_psql manual__2023-10-22T08:38:23.431099+00:00 [running]> on host 90cf752d844d
[2023-10-22T08:38:34.637+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='kel11' AIRFLOW_CTX_DAG_ID='etl_day18' AIRFLOW_CTX_TASK_ID='fetch_data_from_psql' AIRFLOW_CTX_EXECUTION_DATE='2023-10-22T08:38:23.431099+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-10-22T08:38:23.431099+00:00'
[2023-10-22T08:38:34.653+0000] {base.py:73} INFO - Using connection ID 'postgresql_db' for task execution.
[2023-10-22T08:38:37.576+0000] {warnings.py:109} WARNING - /home/***/.local/lib/python3.8/site-packages/***/providers/common/sql/hooks/sql.py:219: UserWarning: pandas only supports SQLAlchemy connectable (engine/connection) or database string URI or sqlite3 DBAPI2 connection. Other DBAPI2 objects are not tested. Please consider using SQLAlchemy.
  return psql.read_sql(sql, con=conn, params=parameters, **kwargs)

[2023-10-22T08:38:37.670+0000] {logging_mixin.py:151} INFO -                       location country  ...           unit          lastUpdated
0   Jakarta-Selatan-CasaGrande      ID  ...  particles/cm³  2023-09-21 14:43:32
1   Jakarta-Selatan-CasaGrande      ID  ...  particles/cm³  2023-09-21 14:43:32
2   Jakarta-Selatan-CasaGrande      ID  ...  particles/cm³  2023-09-21 14:43:32
3   Jakarta-Selatan-CasaGrande      ID  ...            iaq  2023-09-21 14:43:32
4   Jakarta-Selatan-CasaGrande      ID  ...  particles/cm³  2023-09-21 14:43:32
5   Jakarta-Selatan-CasaGrande      ID  ...          µg/m³  2023-09-21 14:43:32
6   Jakarta-Selatan-CasaGrande      ID  ...  particles/cm³  2023-09-21 14:43:32
7   Jakarta-Selatan-CasaGrande      ID  ...             mb  2023-09-21 14:43:32
8   Jakarta-Selatan-CasaGrande      ID  ...          µg/m³  2023-09-21 14:43:32
9   Jakarta-Selatan-CasaGrande      ID  ...  particles/cm³  2023-09-21 14:43:32
10  Jakarta-Selatan-CasaGrande      ID  ...          µg/m³  2023-09-21 14:43:32
11  Jakarta-Selatan-CasaGrande      ID  ...              %  2023-09-21 14:43:32
12  Jakarta-Selatan-CasaGrande      ID  ...              f  2023-09-21 14:43:32
13  Jakarta-Selatan-CasaGrande      ID  ...  particles/cm³  2023-09-21 14:43:32

[14 rows x 7 columns]
[2023-10-22T08:38:37.705+0000] {python.py:194} INFO - Done. Returned value was: None
[2023-10-22T08:38:37.716+0000] {taskinstance.py:1398} INFO - Marking task as SUCCESS. dag_id=etl_day18, task_id=fetch_data_from_psql, execution_date=20231022T083823, start_date=20231022T083834, end_date=20231022T083837
[2023-10-22T08:38:37.750+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-10-22T08:38:37.786+0000] {taskinstance.py:2776} INFO - 1 downstream tasks scheduled from follow-on schedule check
